bin/idl.js

```diff
@@ -961,9 +961,9 @@ function toString() {
     function toTableEntry(remoteKey) {
         var remote = self.remotes[remoteKey];
         var local = self.localRemotes[remoteKey];
-        var localTime = local && local.time || '';
+        var localTime = local && local.time || 0;
         var age = '-';
-        if (localTime && localTime.length > 0) {
```

Comment:
This can be simplified to just the `>` expression. I’ll ammend. Thanks for the diff.